client/command/processes/ps.go

```diff
@@ -113,22 +113,53 @@ func PsCmd(cmd *cobra.Command, con *console.SliverClient, args []string) {
 	if session == nil && beacon == nil {
 		return
 	}
+
+	ownerFilter, _ := cmd.Flags().GetString("owner")
+	cmdLine, _ := cmd.Flags().GetBool("print-cmdline")
+	tree, _ := cmd.Flags().GetBool("tree")
+	fullInfo, _ := cmd.Flags().GetBool("full")
+
+	if tree {
```

Comment:
Correct me if I'm wrong, but you're not checking `fullInfo` here.

Reply:
Good catch! Thank you! :grin: